src/errors/ApiError.ts

```diff
@@ -97,11 +96,11 @@ export default class ApiError extends Error {
    */
   public static createFromResponse(response: AxiosResponse): ApiError {
     return new ApiError(
-      get(response, 'data.detail', 'Received an error without a message'),
-      get(response, 'data.title'),
-      get(response, 'data.status'),
-      get(response, 'data.field'),
-      cloneDeep(get(response, 'data._links')),
+      response.data.detail ?? 'Received an error without a message',
+      response.data.title,
+      response.data.status,
+      response.data.field,
+      response.data._links,
```

Comment:
`data._links` is no longer cloned, but I guess that's ok? Theoretically someone could mutate a `Url` obtained through `getLink`.

Reply:
The cloning behaviour was introduced in e24d9e531983596ebfdc12a035164df0efe5d9e7 (February '19), I'm not entirely sure for what reason.

`response.data` is a fresh object coming from axios, and it is only consumed here. So whether the original is used or it is cloned first doesn't make any difference I can imagine.

Perhaps the idea was indeed to prevent someone from doing `getLink` and then altering the returned object. However, to prevent that, the cloning would have to happen inside `getLink``.

For as far as I can tell, not cloning does not result in a noticeable or measurable change.